src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -70,13 +70,14 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
     return <BaseSystemChat item={item} index={index} />;
   }
 
-  if (item?.isMe)
+  if (item?.isMe) {
     return (
       <ChatItemMyTextV2
         key={index}
         avatar={handleAvatar()}
         isContinuous={item?.isContinuous}
         message={item?.message}
+        attachments={item?.attachmentJson}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ChatItemMyTextV2` component now accepts an `attachments` prop. Ensure that this component is properly handling the `attachments` prop and that it doesn't break if `item?.attachmentJson` is `null` or an unexpected format.

```diff
  if (item?.isMe) {
     return (
       <ChatItemMyTextV2
         key={index}
         avatar={handleAvatar()}
         isContinuous={item?.isContinuous}
         message={item?.message}
+        attachments={item?.attachmentJson}
         time={calculateTime(item?.updatedAt, true)}
         username={handleUserName(item)}
         status={item?.status as ChatStatus}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->